docker.go

```diff
@@ -170,7 +170,10 @@ func DockerImages(args ...string) (string, string, int) {
 
 // DockerRmi removes one or more images
 func DockerRmi(args ...string) (string, string, int) {
-	return runDockerCommand("rmi", args...)
+	// docker takes more than 5 seconds to remove an image, it depends
+	// of the image size and this operation does not involve to the
+	// runtime, 15 seconds should be enough to complete this task
+	return runDockerCommandWithTimeout(15, "rmi", args...)
```

Comment:
I'd rather this was defined at the top of the file as a variable to make it stand out (maybe even also allow the value to be overridden using an env var?)